docs/paper/dev/api/command-api/arguments/registry.mdx

```diff
@@ -0,0 +1,269 @@
+---
+slug: /dev/command-api/arguments/registry
+description: Documentation for arguments retrieving registry values.
+---
+
+import ResourceAttributes from "./assets/registry-arguments/resource-attributes.mp4";
+import ResourceBannerPattern from "./assets/registry-arguments/resource-banner-pattern.mp4";
+import ResourceBiome from "./assets/registry-arguments/resource-biome.mp4";
+import ResourceBlock from "./assets/registry-arguments/resource-block.mp4";
+import ResourceCatVariant from "./assets/registry-arguments/resource-cat-variant.mp4";
+import ResourceDamageType from "./assets/registry-arguments/resource-damage-type.mp4";
+import ResourceDataComponentType from "./assets/registry-arguments/resource-data-component-type.mp4";
+import ResourceFluid from "./assets/registry-arguments/resource-fluid.mp4";
+import ResourceFrogVariant from "./assets/registry-arguments/resource-frog-variant.mp4";
+import ResourceGameEvent from "./assets/registry-arguments/resource-game-event.mp4";
+import ResourceInstrument from "./assets/registry-arguments/resource-instrument.mp4";
+import ResourceItem from "./assets/registry-arguments/resource-item.mp4";
+import ResourceJukeboxSong from "./assets/registry-arguments/resource-jukebox-song.mp4";
+import ResourceMapDecorationType from "./assets/registry-arguments/resource-map-decoration-type.mp4";
+import ResourceMemoryModuleType from "./assets/registry-arguments/resource-memory-module-type.mp4";
+import ResourceMenu from "./assets/registry-arguments/resource-menu.mp4";
+import ResourceMobEffect from "./assets/registry-arguments/resource-mob-effect.mp4";
+import ResourcePaintingVariant from "./assets/registry-arguments/resource-painting-variant.mp4";
+import ResourceSoundEvent from "./assets/registry-arguments/resource-sound-event.mp4";
+import ResourceStructureType from "./assets/registry-arguments/resource-structure-type.mp4";
+import ResourceVillagerProfession from "./assets/registry-arguments/resource-villager-profession.mp4";
+import ResourceVillagerType from "./assets/registry-arguments/resource-villager-type.mp4";
+import ResourceWolfVariant from "./assets/registry-arguments/resource-wolf-variant.mp4";
+
+import ResourceKeyItem from "./assets/registry-arguments/resourcekey-item.mp4";
+import EnchantsRegistry from "./assets/registry-arguments/enchants-registry.mp4";
+
+# Registry Arguments
+Registries in Minecraft hold all sort of information - possible item materials, block materials, enchantments, potion effects, ... and more!
```

Comment:
You shouldn't mention material just item/block or item/block type. The material enum will disappear